client/src/pages/NewTalk.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useContext } from 'react';
+import { navigate } from "@reach/router"
+
+import MaxLengthText from '../components/MaxLengthText';
+import config from '../utils/config';
+import UserContext from '../components/UserContext';
+import Title from '../components/Title';
+
+const NewTalk = () => {
+
+  const [ title, setTitle ] = useState('');
+  const [ abstract, setAbstract ] = useState('');
+
+  const { user } = useContext(UserContext);
+
+  const submit = async e => {
+    e.preventDefault();
+
+    await fetch(`${config.apiHost}/talks`, {
+      method: 'POST',
+      headers: {
+        'Authorization': 'Bearer ' + user,
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({
+        title,
+        abstract
+      })
+    });
+
+    navigate(`${process.env.PUBLIC_URL}/`);
+  };
+
+  const cancel = () => navigate(`${process.env.PUBLIC_URL}/`);
+
+  return (
+    <form action='#' onSubmit={submit} className='body-container'>
```

Comment:
Few issues with your create talk view. There is no loading/error state, and I can create the same talk twice by clicking during loading. Since we're going full-on a11y in this app, can you use a `fieldset` element here to improve this?